<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inventoryWindow</class>
 <widget class="QDialog" name="inventoryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>500</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblInv">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>411</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>Inventory</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbl_items">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>621</width>
     <height>351</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <column>
    <property name="text">
     <string>Item name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Description</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Weight</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>View Item</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Item name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDesc">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>301</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblValue">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Value:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblWeight">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Weight:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblDamage">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Damage:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblUses">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Uses:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblEffect">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>350</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Effect:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblDefense">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>350</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Defense:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShowTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Inventory</string>
   </property>
  </widget>
  <zorder>tbl_items</zorder>
  <zorder>btnExit</zorder>
  <zorder>lblInv</zorder>
  <zorder>lblName</zorder>
  <zorder>lblDesc</zorder>
  <zorder>lblValue</zorder>
  <zorder>lblWeight</zorder>
  <zorder>lblDamage</zorder>
  <zorder>lblUses</zorder>
  <zorder>lblEffect</zorder>
  <zorder>lblDefense</zorder>
  <zorder>btnShowTable</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
